.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -2,88 +2,35 @@
+it('allows adding a comment to sample blogpost', () => {
   cy.visit('/');
   cy.get('.wp-block-query');
   cy.get('.wp-block-post-title a').click();
-  cy.fixture('helloworld').then((hw) => {
-    cy.contains('.wp-block-post-title', hw.title);
-    cy.contains('.wp-block-post-content', hw.content);
-  });
-
-  cy.get('.wp-block-post-title').click();
-  cy.get('.commentlist').should('have.length', 1);
-  cy.fixture('helloworld').then((hw) => {
-    cy.get('.comment-author').should('contain.text', hw.commenter);
-
-    cy.fixture('helloworld').then((hw) => {
-      cy.get('#comment').type(hw.comment).should('have.value', hw.comment);
-    });
-
-    cy.fixture('user').then((user) => {
-      cy.get('#author').type(user.username).should('have.value', user.username);
-      cy.get('#email').type(user.email).should('have.value', user.email);
-    });
-
-    cy.get('#submit').click();
+  cy.fixture('posts').then((post) => {
+    cy.contains('.wp-block-post-title', post.title);
+    cy.contains('.wp-block-post-content', post.content);
+    cy.get('.wp-block-post-title').click();
+    cy.get('.commentlist').should('have.length', 1);
+    cy.contains('.comment-author', post.commenter);
+    cy.get('#comment').type(`${post.comment}.${random}`);
   });
-});
-
-it('disallows login to an invalid user', () => {
-  cy.clearCookies();
-  cy.visit('/wp-login.php');
-  cy.fixture('user').then((user) => {
-    cy.get('#user_login')
-      .type(user.username)
-      .should('have.value', user.username); // Since login can be flaky, leaving additional asserts
-    cy.get('#user_pass')
-      .type(user.password)
-      .should('have.value', user.password);
-  });
-  cy.get('#wp-submit').click();
-  cy.fixture('user').then((user) => {
-    cy.get('#login_error').should(
-      'contain.text',
-      `The username ${user.username} is not registered on this site.`
-    );
-  });
-});
-
-it('disallows login to a valid user with wrong password', () => {
-  cy.clearCookies();
-  cy.visit('/wp-login.php');
-  cy.get('#user_login')
-    .type(Cypress.env('username'))
-    .should('have.value', Cypress.env('username'));
-  cy.fixture('user').then((user) => {
-    cy.get('#user_pass')
-      .type(user.password)
-      .should('have.value', user.password);
+  cy.fixture('users').then((user) => {
+    cy.get('#author').type(user.userName);
+    cy.get('#email').type(user.userEmail);
   });
-  cy.get('#wp-submit').click();
-  cy.get('#login_error').should(
-    'contain.text',
-    `Error: The password you entered for the username ${Cypress.env(
-      'username'
-    )} is incorrect. Lost your password?`
-  );
-});
 
-it('checks the blog name and user email configuration', () => {
-  cy.login();
-  cy.visit('/wp-admin/options-general.php');
-  cy.get('#new_admin_email').should(
-    'have.value',
-    Cypress.env('wordpressEmail')
+  cy.get('#submit').click();
+  cy.contains(
+    '.comment-awaiting-moderation',
+    'your comment will be visible after it has been approved'
```

Comment:
I don't you need a CSS class selector here. If the second string appears on screen, it should be enough to know it worked :)

Reply:
Fixed.